### PR TITLE
updated tool URLs

### PR DESCRIPTION
**Pull request by bagnacan**
- Tool URLs link to https://rna.usegalaxy.eu/...
- DOI links point to https://doi.org/...
- Removed tool URLs from entries that represent tool suites (SAM/BAM/deep tools, etc)
- [MEA](https://rna.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/mea/mea), [CMCompare](https://rna.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/cmcv/cmcv), [RNAammer](https://rna.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/lionelguy/rnammer) links are broken. We should include them in usegalaxy.eu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,138 +149,153 @@ In this section we list all tools that have been integrated in the RNA workbench
...
 
 <p align="right"><a href="#top">&#x25B2; back to top</a></p>
 
 ## Ribosome profiling
 
 Tool | Description | Reference
 --- | --- | ---
-[RiboTaper](https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/) | An analysis pipeline for Ribo-Seq experiments, exploiting the triplet periodicity of ribosomal footprints to call translated regions | [Calviello et al. 2015](https://doi.org/10.1038/nmeth.3688)
-[Trinity](https://github.com/trinityrnaseq/trinityrnaseq/wiki) | A tool to assemble transcript sequences from Illumina RNA-Seq data | [Grabherr et al. 2011](https://doi.org/10.1038/nbt.1883)
+RiboTaper | An analysis pipeline for Ribo-Seq experiments, exploiting the triplet periodicity of ribosomal footprints to call translated regions | [Calviello et al. 2015](https://doi.org/10.1038/nmeth.3688)
```
Comment (bgruening) [inline README.md]: I think this is installed on EU. This is a 3 step tool.

Comment (bagnacan) [reply README.md]: Yes, and the same holds for DR. Disco.
Shall I link their first steps?

Comment (bgruening) [reply README.md]: Is it installed? I have not found it.

Comment (bagnacan) [reply README.md]: @bgruening, no, it is missing.
I'm creating a PR to include the following tools in usegalaxy.eu:
- Dr. Disco
- CMCompare
- MEA
- RNAammer
this way, users will see the workbench as a selected subset of the RNA tools found on rna.usegalaxy.eu

**Pull request merged**